File: hackerearth/TerribleChandu.cpp
/*
Contest Name: Code Monk(Array& Strings)
Date: Jun 4, 2015
*/

#include <iostream>
using namespace std;

string reverseString(string inputString){
	string outputstring;
	for(int i=inputString.length();i>=0;i--){
		outputstring=outputstring+inputString[i];
	}
	return outputstring;
}

int main()
{
    int T;
    cin>>T;
    string outputstrings[10];
    int i=0;
    while(T--){
    	string inputString;
    	cin>>inputString;
    	outputstrings[i]=reverseString(inputString);
    	i++;
    }

    for(int j=0;j<i;j++){
    	cout<<outputstrings[j]<<endl;
    }
    	
    return 0;
}

File: hackerearth/TerribleChandu.java
/*
Contest Name: Code Monk(Array& Strings)
Date: Jun 4, 2015
*/

import java.io.*;
import java.util.*;
class TestClass {
	public static String reverseString(String s){
		String output="";
		for(int i=s.length()-1;i>=0;i--){
			output=output+s.charAt(i);
			
		}
		return output;
	}
    public static void main(String args[] ) throws Exception {
       
		Scanner in=new Scanner(System.in);
		int T=in.nextInt();
		while(T>0){
			String inputString=in.next();
			System.out.println(reverseString(inputString));
			T--;
		}
        
    }
}